update_version.py

```diff
@@ -1,43 +1,83 @@
 #!/usr/bin/env python3
-# Usage: ./update_version.py <MAJOR>.<MINOR>.<MICRO> [<RC version>]
+# Usage: ./update_version.py
 #
 # Example:
-# ./update_version.py 3.7.1 2
-#   => Version will become 3.7.1-rc-2 (beta)
-# ./update_version.py 3.7.1
-#   => Version will become 3.7.1 (stable)
-
+# version.json
+# {
+#     "main": {
+#         "protoc_version": "21-dev",
+#         ...
+#         "languages": {
+#             "cpp": "3.21-dev",
+#             "java": "4.21-dev",
+#             ...
+#         }
+#     }
+# }
+# ./update_version.py
```

Comment:
This will be included in the internal release_open_source script (will send out CL for review soon), which will update version.json to reflect a new micro/macro release or RC depending on the step in the release process. This is because this requires understanding which step in the release we are in and the git branch we are currently part of. I've attempted to leave update_version.py agnostic.

Will add this reasoning to the design doc.

Reply:
Note design was updated with a section on this, and CL sent for corresponding chnages to the release script